### ISSUE TITLE
Spacer: Missing default dimensions

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#6777]
**Description**
The spacer is a 1x1 pixel transparent image.  If you have no defaults set for height and width you are saying you want a 1x1 pixel spacer.  However this causes a Google Lighthouse error.

Error:
```xml
<p:spacer/>
```

OK:
```xml
<p:spacer height="1" width="1"/>
```

![image](https://user-images.githubusercontent.com/4399574/103809240-3189e780-5027-11eb-9387-377dd07801a8.png)

See: https://web.dev/optimize-cls/?utm_source=lighthouse&utm_medium=devtools#images-without-dimensions

Basically the browser wants to know the size of an image up front for performance vs loading the image and figuring it out.

**Comment by tandraschko** [comment]
is space a new component? TBH i dont like such components as its the same as "div width=1px height=1px" which will have a much better perf

**Comment by melloware** [comment]
Spacer has been there since PF1!

**Comment by melloware** [comment]
Also spacer comes in handy when you need a PF component like in PanelGrid where a DIV does not work.
```xml
<p:panelGrid columns="2">
    <h:outputText  value="1" />
    <p:spacer />

    <h:outputText value="3" />
    <h:outputText value="4" />
</p:panelGrid>
```

**Comment by tandraschko** [comment]
oh ok :D Then +1.

Maybe we should then also create a issue to support all different size units? currently "1" means px, what about em etc.?

about your example: yep div doesnt work but you can jsf:id to make it work.

**Comment by melloware** [comment]
well the default image is a 1pxx1px transparent pixel the smallest image you can make as the default.  But the Height and width are strings so you can do.

```xml
 <p:spacer height="2rem" width="3rem" />
```
